db/flush_job_test.cc

```diff
@@ -181,8 +181,8 @@ TEST_F(FlushJobTest, NonEmpty) {
   // Note: the first two blob references will not be considered when resolving
   // the oldest blob file referenced (the first one is inlined TTL, while the
   // second one is TTL and thus points to a TTL blob file).
-  constexpr std::array<uint64_t, 6> blob_file_numbers{
-      kInvalidBlobFileNumber, 5, 103, 17, 102, 101};
+  constexpr std::array<uint64_t, 6> blob_file_numbers{{
```

Comment:
Could you try `constexpr std::array<uint64_t, 6> blob_file_numbers = {kInvalidBlobFileNumber, 5, 103, 17, 102, 101};` ? I think that would work too and it looks nicer :)

Reply:
I thought = for initialization was arcane / taboo. ¯\_(ツ)_/¯

Reply:
Well, the nice (and completely legit) way of doing it is what's on master... Since that doesn't work with this compiler, I say let's pick the least ugly workaround :)

Reply:
Hm, it seems that this version of `clang` also chokes on the `=` version that was standard even before CWG 1270. Double braces it is then... :(